doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -5,162 +5,256 @@ This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
 ifdef::DM[]
 
-== Upgrading
-
-.After upgrading to {PRODUCT} 7.3.0, rules do not fire.
-
-Issue: If you upgrade to {PRODUCT} 7.3.0, rules do not fire because of a change to packages in the `kmodule.xml` file. For more information, see
-https://access.redhat.com/solutions/4014541[Rules don't fire after upgrading to RHDM 7.3.0].
 endif::[]
 
 == {CENTRAL}
-ifdef::DM[]
-.All line separators in the right-hand side of rules in the spreadsheet are escaped [https://issues.jboss.org/browse/RHDM-903[RHDM-903]]
 
-Issue: If you have multiple lines in the *ACTION* column of a spreadsheet, all line separators are converted to `\n`. This conversion causes a build error.
+.You cannot add a new group in {CENTRAL} integrated with RH-SSO [https://issues.jboss.org/browse/RHPAM-2036[RHPAM-2036]]
+
+Issue: The following exception occurs when you try to add a new group in {CENTRAL} if {CENTRAL} is integrated with RH-SSO 7.3.0:
+[source]
+----
+java.lang.NoSuchMethodError:org.keycloak.representations.idm.RoleRepresentation.setScopeParamRequired(Ljava/lang/Boolean;)
+----
 
 Steps to reproduce:
 
-. Add more than one line to a row in the in *ACTION* column of a spreadsheet.
-. Build the spreadsheet with {PRODUCT}.
+. Open {CENTRAL}.
+. Open the *Admin* page.
+. Open *Groups*.
+. Try to create a new group.
 
-Expected result: The spreadsheet builds successfully.
 
-Actual result: The build fails with errors similar to `line 1:15 no viable alternative at character '\'`.
+Expected result: The new group is created.
 
-Workaround: Ensure that rows in the *ACTION* column of a spreadsheet do not contain more than one line in each row.
-endif::[]
+Actual result: A dialog box with the listed exception appears.
+
+Workaround: Add the group in RH-SSO instead of adding it in {CENTRAL}.
 
-.You cannot upload a spreadsheet that contains a large decision table [https://issues.jboss.org/browse/RHDM-488[RHDM-488]]
+.Human task comments are not retained when a process instance is finished
+//[https://issues.jboss.org/browse/JBPM-8556[JBPM-8556]]
 
-Issue: If you try to upload a spreadsheet that contains a large decision table into {CENTRAL}, an out-of-memory error appears in the `error.log` file.
+Issue: Human task comments are retained and visible only while the respective process instance is running. When the process instance is completed or aborted, comments from human tasks disappear even though the tasks are still visible in {CENTRAL}.
 
 Steps to reproduce:
 
-. Create a spreadsheet that contains a large decision table.
-. Create a new project in {CENTRAL}.
-. Click *Add Asset* and then select *Spreadsheet Decision Table*.
-. Attach the spreadsheet that contains the large decision table.
-. Review the server `error.log` file.
+. Start a process instance with two consecutive human tasks.
+. Comment on the first human task.
+. Claim, start, and complete the first task.
+. Check the log file to confirm that the comments are retained for the completed task.
```

Comment:
It is not log file (here and in 'Excpected' and 'Actual' results below), but rather a Tab in UI, can we refer to it as Comments Tab (of a Human Task)?

Reply:
Fixed